src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
+			source: 'auto'    // 'auto|labels'
 		}
 	};
 
+	function buildLookupTable(ticks, min, max, linear) {
+		var ilen = ticks.length;
+		var table = [];
+		var i, prev, curr, next, decimal;
+
+		if (ilen === 0) {
+			return table;
+		}
+
+		if (ticks[0] !== min) {
+			table.push({time: min, decimal: 0});
+		}
+
+		for (i=0, ilen=ticks.length; i<ilen; ++i) {
```

Comment:
No, better like that for minification, gather all var declarations in the same block.